Docker setup doesn't work; `make install` should be after `make jekyll`?

Docker setup doesn't work out of the box:
```bash
$ make docker
sudo docker build -t badssl .
...
Sending build context to Docker daemon 7.457 MB
Step 1 : FROM ubuntu:16.04
 ---> 17b6a9e179d7
Step 2 : MAINTAINER April King <april@twoevils.org>
 ---> Using cache
 ---> 496c4e159766
Step 3 : EXPOSE 80 443
 ---> Using cache
 ---> 43f7090ccd2a
Step 4 : RUN apt-get update && apt-get install -y     build-essential     git     make     nginx     ruby     ruby-dev
 ---> Using cache
 ---> 9bbe69580e2d
Step 5 : RUN gem install jekyll
 ---> Using cache
 ---> b24fb3692d8b
Step 6 : ADD . badssl.com
 ---> 79911ecdaab9
Removing intermediate container c7935c0d8c5f
Step 7 : WORKDIR badssl.com
 ---> Running in 5cb3d08bd206
 ---> 7dd27573c74c
Removing intermediate container 5cb3d08bd206
Step 8 : RUN make install
 ---> Running in 1fd64c9b4899
./_site/certs/cert-generator/cert-generator.sh y
make: ./_site/certs/cert-generator/cert-generator.sh: Command not found
make: *** [keys] Error 127
```

It works if I change the Dockerfile to run `make jekyll` after `make install`, but not sure if that's the right fix or not. @Hoverbear

--- a/Dockerfile
+++ b/Dockerfile
@@ -14,8 +14,8 @@ RUN gem install jekyll
 # Install badssl.com
 ADD . badssl.com
 WORKDIR badssl.com
-RUN make install
 RUN make jekyll
+RUN make install
 
 # Start things up!
 CMD nginx && tail -f /var/log/nginx/access.log /var/log/nginx/error.log

--- a/Makefile
+++ b/Makefile
@@ -17,7 +17,9 @@ clean:
 
 .PHONY: keys
 keys:
+	ln -s ../certs _site/common/certs # Create symlink to certs directory
 	./_site/certs/cert-generator/cert-generator.sh y
+	./_site/certs/cert-generator/cert-self-signed-symlink-generator.sh # Generate symlinks to self-signed for everything that doesn't exist in certs
 
 .PHONY: install-keys
 install-keys:
@@ -38,8 +40,6 @@ install: keys install-keys link
 .PHONY: jekyll
 jekyll:
 	DOMAIN="${SITE}" HTTP_DOMAIN="http.${SITE}" jekyll build
-	ln -s ../certs _site/common/certs # Create symlink to certs directory
-	./_site/certs/cert-generator/cert-self-signed-symlink-generator.sh # Generate symlinks to self-signed for everything that doesn't exist in certs
 
 .PHONY: docker
 docker:

--- a/certs/cert-generator/cert-generator.sh
+++ b/certs/cert-generator/cert-generator.sh
@@ -18,6 +18,9 @@ else
   echo
 fi
 
+# Ensure the directory exists, otherwise we can't populate certificates.
+mkdir -p ../self-signed
+
 if [[ $regen =~ ^[Yy]$ ]]; then
   echo "Generating BadSSL.com Root Certificate Authority"
   openssl req -new -x509 -days 7300 \

